core/src/kiso.rs

```diff
@@ -29,7 +29,7 @@ impl KisoHandle {
     /// Spawn a new actor
     pub fn new(state: Config) -> Self {
         let (actor_sender, actor_receiver) = mpsc::channel(DEFAULT_CHANNEL_SIZE);
-        let (log_level_update, _) = watch::channel(state.logger.level);
+        let (log_level_update, _) = watch::channel(state.logger.level.clone());
```

Comment:
not sure if this is appropriate place to comment. What happens in the following scenario:
1. We set LOG_LEVEL="error,iroha_core=info"
2. We dynamically update LOG_LEVEL="debug"

what is `iroha_core` log level now?

Reply:
new `LOG_LEVEL` completely overwrites previous one, so `iroha_core` new log level is `debug` as for every other module

Reply:
I thought so, and I find this behavior unexpected

Reply:
ok, this wasn't a good example:
1. We set LOG_LEVEL="error,iroha_core=info"
2. We dynamically update LOG_LEVEL="iroha_core=debug"

will this affect only `iroha_core`, or will every module be set to default?

Reply:
I think that dynamic update should be granular, what do you think? But it's fine for me to merge this PR

Reply:
for granular updates i think we should use different endpoint like `PATCH /configuration` instead of `POST /configuration`

Reply:
> I think that dynamic update should be granular

I find the current behavior (updating the full filter string, overwriting whatever was there before, as if you set a new configuration value) more intuitive (and simpler to implement).

If the worst comes to the worst, one can still get the configuration, add the necessary filters and set it back